assets/css/ui.notice.scss

```diff
@@ -0,0 +1,244 @@
+.jm-notice,
```

Comment:
Looks like we have a mix of prefixes: `wpjm`, `jm`, `job_manager`, `job-manager`, `wp-job-manager` (plus no prefix 😄). 
Since we are starting to brand the plugin just as 'Job Manager', especially when already inside WordPress, I'd go with the `jm` one for the new styles, it's short and easy to remember and say :)